docs/workloads/azure/data/etl_pipelines/ingest_data_azure.md

```diff
@@ -66,19 +71,22 @@ is triggered in Data Factory, and all entities will be ingested. To disable a pa
 source or entity without removing it, you can set `"enabled": false` – these will be ignored by
 the Data Factory pipeline.
 
```

Comment:
Could we also get the breakdown of how these values are used to build our query for ingestion?

```suggestion
The configuration values are used to drive the following command for data ingestion:
select COLUMNS from TABLE ....etc etc
```

Reply:
Added a section explaining this